NOTICE.txt

```diff
@@ -0,0 +1,985 @@
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties. Microsoft makes certain
+open source code available at http://3rdpartysource.microsoft.com, or you may
+send a check or money order for US $5.00, including the product name, the open
+source component name, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the
+extent required to debug changes to any libraries licensed under the GNU Lesser
+General Public License.
+
+
+
+License notices for BouncyCastle.Crypto.dll
```

Comment:
I don't think we need to include the bouncy castle notice as it's not shipping.

Reply:
Also here you have 

License notice**s** instead of
License notice

Reply:
Typo is solved